portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jennifer's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <header>
+    <h1>JENNIFER JIMENEZ</h1>
+    <div class="logos">
+      <a href="https://www.linkedin.com/in/jen-jimenez" target="_blank"> 
+        <img src="/images/linkedin.png" id="logo"/>
+      </a>
+      <a href="https://github.com/jennifer-jimenez" target="_blank"> 
+        <img src="/images/git.png" id="logo"/>
+      </a>
+      <a href="https://www.instagram.com/prettyoddjen/" target="_blank"> 
+        <img src="/images/ig.png" id="logo"/>
+      </a>
+    </div>
+  </header>
+  <nav>
+    <ul class="navbar">
+      <li><a href="index.html" class="nav_link">Home</a></li>
+       <li><a href="projects.html" class="nav_link"><u>Projects</u></a></li>
+      <li><a href="gallery.html" class="nav_link">Gallery</a></li>
+    </ul>
+  </nav>
+
+  <body>
+  <div id="projects">
+    <h2>Schedule Pro</h2>
+    <p>
+      Schedule Pro is a web app that takes a user’s weekly non-negotiable scheduled tasks (e.g. class, work) as 
+      well as tasks that can be completed any time (e.g. going to the store) and optimizes their route based on 
+      distance. Distance is calculated using data from Google Maps Distance Matrix API. The web app was created 
+      using the Flask microframework (incorporates Python, HTML/CSS, Jinja, and SQL).
+    </p>
+    <iframe id="video" src="https://www.youtube.com/embed/3tKCzG9TkoY" title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
nit: This line is a bit long. You could probably break it at each attribute, as discussed here: https://stackoverflow.com/questions/34873362/what-is-the-best-way-of-formatting-a-lot-of-html-attributes-in-angular-html-code